llvm/test/tools/sycl-post-link/device-requirements/fixed-target.ll

```diff
@@ -0,0 +1,32 @@
+; RUN: sycl-post-link -split=auto %s -o %t.files.table
+; RUN: FileCheck %s -input-file=%t.files_0.prop
+
+; CHECK: [SYCL/device requirements]
+; CHECK: fixed_target=2|gAAAAAAAAAQAAAAA
```

Comment:
Wanted to quickly check if this is intended (gAAAAAAAAQAAAAA). 
Thanks

Reply:
Byte array properties are emitted in form of Base64-encoded string, so yes, this weird string is likely intended here to at least check that further changes to the pass don't accidentally change the content of this property.

However, I do agree that we should implement some mechanism which will allow us to write human-readable tests for byte array properties